lib/instruments.js

```diff
@@ -278,7 +278,7 @@ Instruments.prototype.launchAndKill = function (condition, cb) {
   cb = _.once(cb);
   var warnlevel = 10; // if we pass 10 attempts to kill but fail, log a warning
   logger.info("Launching instruments briefly then killing it");
-  this.setInstrumentsPath(function (err) {
```

Comment:
needed to replace this call to 'setInstrumentsPath' with 'configure', since the TraceTemplate needs to get set here too.